src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -703,6 +703,7 @@ object SampleWdl {
         |workflow two_step {
         |  String flags_suffix
         |  String flags = "-" + flags_suffix
+        |  String static_string = "foobarbaz"
```

Comment:
Prior to this fix, it'd consider static_string an input to the workflow.  the DeclarationWorkflowSpec asserts the workflow inputs so adding this to the WDL does indeed test the right path.